src/content/docs/en/recipes/add-yaml-support.mdx

```diff
@@ -34,3 +34,15 @@ Astro builds on top of Vite, and supports both Vite and Rollup plugins. This rec
     ```js
     import yml from './data.yml';
     ```
+
+    :::note
+    While you can now import YAML data in your Astro project, your editor will not provide types for the imported data. To add types, create a `*.d.ts` file in the `src` directory of your project and add the following:
+    ```ts
```

Comment:
```suggestion
    ```ts title="src/*.d.ts"
```
Is there any reason not to do this? (Though, if `*` is a wildcard, I'd prefer an actual file name here someone might use.)

Reply:
We could set it to `files.d.ts` to show it's intentions?
```suggestion
    ```ts title="src/files.d.ts"
```